test/functional/p2p_tx_download.py

```diff
@@ -265,7 +265,8 @@ def test_rejects_filter_reset(self):
         self.generate(self.wallet, 1, sync_fun=self.no_op)
         peer.sync_with_ping()
         peer.send_and_ping(msg_inv([CInv(t=MSG_WTX, h=int(low_fee_tx['wtxid'], 16))]))
-        node.bumpmocktime(MAX_GETDATA_INBOUND_WAIT)
+        node.setmocktime(int(time.time()))
+        node.bumpmocktime(MAX_GETDATA_INBOUND_WAIT + 300)
```

Comment:
I haven't been able to reproduce this issue and am mostly guessing here, but if it's a `setmocktime` problem, perhaps just do a `node.setmocktime(0)` to reset it a few lines up (shouldn't impact the test except maybe make it take 2sec longer)?

Reply:
I initially had has only used the `node.setmocktime(0)` in that place and that succeeded when I temporarily pushed it in an unrelated PR but then here [it failed when I opened this one](https://github.com/bitcoin/bitcoin/runs/24062801267). After adding the +300 it succeeded, but yeah, this may have been just luck again. I will push a version without +300 and and the `node.setmocktime(0)` moved higher up.

I just saw that the Re-Run button also appears for CI jobs that succeeded, I will try to re-rerun this version a few times manually so we can have bit more confidence before merging.

Reply:
Had another failure, so I will add the +300 back and see if that is a permanent fix.